### PR TITLE
DOCS-248: add details

### PR DESCRIPTION
**Pull request by kay-kim**
More of a rough guide -- not too comprehensive - as ultimately, the user-interface itself should strive to be intuitive.

https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/
https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/login/
https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/update-profile/
https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/manage-files/
https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/manage-contacts/
https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/manage-bookmarks/

For now, added screenshots.  Usually, I don't use screenshots because of the maintenance issues with ensuring that the screenshots are up-to-date -- but at this stage of the product, I think it should be okay.

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -16,67 +16,6 @@ With Inrupt `PodBrowser`_, you can
```
Comment (kay-kim) [inline source/index.rst:16]: https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/

File: source/login.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
+=======================
+Login and Authorization
+=======================
```
Comment (kay-kim) [inline source/login.rst:3]: https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/login/

File: source/manage-bookmarks.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
+====================================
+View Saved Bookmarks: Bookmarks View
+====================================
```
Comment (kay-kim) [inline source/manage-bookmarks.rst:3]: https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/manage-bookmarks/

File: source/manage-contacts.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
+==============================
+Manage Contacts: Contacts View
+==============================
```
Comment (kay-kim) [inline source/manage-contacts.rst:3]: https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/manage-contacts/

File: source/manage-files.rst [hunk 5]
```diff
@@ -0,0 +1,191 @@
+=======================
+Manage Data: Files View
+=======================
```
Comment (kay-kim) [inline source/manage-files.rst:3]: https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/manage-files/

File: source/update-profile.rst [hunk 6]
```diff
@@ -0,0 +1,19 @@
+==============
+Update Profile
+==============
```
Comment (kay-kim) [inline source/update-profile.rst:3]: https://docs-inrupt-private-git-stage-podbrowser.kay-kim.vercel.app/update-profile/

File: source/manage-files.rst [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+In the :guilabel:`Files` view, click on a folder to navigate
+into the folder.
+
+.. _view-details:
+
+View Details of Folders and Files
+=================================
+
+To view the details of a folder,
```
Comment (brownhoward) [inline source/manage-files.rst]: Formatting seems off here. Should it be a numbered list like other pages?

File: source/manage-files.rst [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+
+   From the folder's Details view:
+
+   - You can :ref:`Delete <delete>` the folder.
+
+   - You can view its Details.
+
+   - You can modify the :ref:`Permissions <modify-permissions>` to the folder.
+
+To view the details of a file/Resource,
```
Comment (brownhoward) [inline source/manage-files.rst]: As above, formatting seems off.

File: source/manage-contacts.rst [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+To manage contacts in PodBrowser, switch to the :guilabel:`Contacts`
```
Comment (brownhoward) [inline source/manage-contacts.rst:5]: Seems repetitive.  Maybe "To manage contacts in PodBrowser, switch to the :guilabel:`Contacts` view by clicking on the Contacts icon on the top menu bar."

File: source/manage-files.rst [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+
+.. _files-view-bookmark:
+
+Bookmark Files and Folders
+==========================
+
+From the Files view, you can bookmark a folder or a file by selecting
+the star icon next to the folder or file. To remove the folder or file
+from your bookmarks, deselect the star icon.
+
```
Comment (brownhoward) [inline source/manage-files.rst:237]: If you want bookmark the folder you are currently viewing:

#. Click the Pod Indicator dropdown just below the Account menu.

#.  Select the :guilabel:`Bookmark Pod` option

File: source/manage-files.rst [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+   * - Resource
+
+     - A Resource refers to a file of any type.
+
+Navigate the Pod
+================
+
+In the :guilabel:`Files` view, click on a folder to navigate
+into the folder.
+
```
Comment (brownhoward) [inline source/manage-files.rst:50]: If you want to navigate directly to a Container in your Pod, or even in another Pod:

#. Click the Pod Indicator dropdown just below the Account menu.

#. Select the :guilabel:`Change Pod` option.

#. Enter the full URL of the Container to which you want to navigate (e.g., `https://<Pod URL/public`).

#. Click :guilabel:`GO`

The contents of the Container will be displayed if you have Read access.

**Review by brownhoward — CHANGES_REQUESTED** [review]
Looked good. As an FYI, Kyra is proposing changes to the designs :-(

File: source/manage-files.rst (hunk 7 above)
Comment (kay-kim) [reply source/manage-files.rst]: Updated.

File: source/manage-files.rst (hunk 8 above)
Comment (kay-kim) [reply source/manage-files.rst]: updated

**Comment by kay-kim** [comment]
Yeah - I figured we'd change the designs of these quite a bit in the early days.  That's okay - I expected frequent updates in the beginning (hence, this isn't particularly in-depth).  Probably in 9 mos or so, we can remove the images; and although the designs might change, at least, by that time, the labels and UI organization will be more stable so that the steps will remain somewhat stable.

**Review by brownhoward — APPROVED** [review]

**Pull request merged**